ios/default.nix

```diff
@@ -353,10 +353,8 @@ nixpkgs.runCommand "${executableName}-app" (rec {
     cp -LR "$(dirname $0)/../${executableName}.app" $tmpdir
     chmod +w "$tmpdir/${executableName}.app"
     mkdir -p "$tmpdir/${executableName}.app/config"
-    cp "$1" "$tmpdir/${executableName}.app/config/route"
 
-    # focus????
-    focus/reflex-platform/scripts/run-in-ios-sim "$tmpdir/${executableName}.app"
+    (cd "$(dirname $0)/../../reflex-platform/scripts" && ./run-in-ios-sim "$tmpdir/${executableName}.app")
```

Comment:
Should we put the "run-in-ios-sim" inside `result/bin` to avoid hardcoding the path?